server/Resources/Resources.resx

```diff
@@ -4757,6 +4757,9 @@ Set to "External URL" to use an API definition that is hosted elsewhere.</value>
   <data name="invalidStartupCommandNodejs" xml:space="preserve">
     <value>Startup commands for NodeJS must use ng, node or pm2</value>
   </data>
+  <data name="invalidStartupCommandAspNetCore" xml:space="preserve">
+    <value>Startup commands for AspNetCore must use dotnet</value>
```

Comment:
Should we say ..must start with dotnet?

Reply:
Again, I tried to maintain consistency with the other message. I'll take suggestions from PM/Sachin over here.

Reply:
Add examples